### ISSUE TITLE
[MAS2.1.1] Unable to use Inspect functionality using keyboard.

### ISSUE THREAD

**Issue by priysn** [Microsoft/BotFramework-Emulator#741]
**Actual Result:**
User is not able to access chat text lines using keyboard and hence not able to inspect them using keyboard.

Expected Result:
User should be able to access chat text lines using keyboard and hence should be able to inspect them using keyboard.

**Repro Steps:**
1. Open the Bot Framework V4 Emulator.
2. Click on open bot and choose the .bot file that is available in the bot folder.
3. Verify that live chat tab is open.
4. Notice 'Inspector' section present in right side.
5. Click on any chat text using mouse and notice that Information related to this chat text line is appearing in the Inspector section.
6. Now try to perform the same using keyboard.

**User Impact:**
Keyboard dependent users won't be able to use the controls which are not accessible using keyboard in any way.

**Test Environment:**
OS: Windows 10
OS Build: 17134.167
OS Version: 1803
Browser: Edge

**Impacted MAS Link**:
[MAS 2.1.1 – Keyboard (36)](https://microsoft.sharepoint.com/:w:/r/teams/msenable/_layouts/15/WopiFrame.aspx?sourcedoc={8492c4eb-c179-40ae-8777-cd044ed725a2})

**Attachment:**

[MAS2.1.1_BotFrameworkEmulator_Chat_Inspector.zip](https://github.com/Microsoft/BotFramework-Emulator/files/2266420/MAS2.1.1_BotFrameworkEmulator_Chat_Inspector.zip)

**Comment by corinagum** [comment]
Reassigned to @compulim since this is a change that needs to be started in WebChat
